tests/integ/modin/frame/test_astype.py

```diff
@@ -102,25 +102,40 @@ def test_astype_from_timestamp_ltz(session, to_dtype):
     "dtype",
     [
         "timedelta64[ns]",
-        {"col1": str, "col2": "timedelta64[ns]"},
+        {"col1": int, "col2": "timedelta64[ns]"},
     ],
 )
 @sql_count_checker(query_count=1)
-def test_astype_int_to_timedelta(dtype):
-    native_df = native_pd.DataFrame(data={"col1": [12345678, 2], "col2": [3, 4]})
+def test_astype_numeric_to_timedelta(dtype):
```

Comment:
Why use this name even though one of the input columns is `[True, False]`?

Reply:
Good catch, let me rename